collect_app/src/main/java/org/odk/collect/android/activities/GoogleDriveActivity.java

```diff
@@ -379,6 +380,7 @@ protected void onListItemClick(ListView l, View v, int position, long id) {
                 createAlertDialog(getString(R.string.no_connection));
```

Comment:
I think you'll want to re-enable the adapter if the network isn't available, otherwise things won't be clickable when the connection comes back.